toplevel/coqargs.ml

```diff
@@ -38,6 +38,8 @@ type color = [`ON | `AUTO | `OFF]
 
 type native_compiler = NativeOff | NativeOn of { ondemand : bool }
 
+type option_set = OptionSet of string option | OptionUnset
```

Comment:
This is a bad name, it reads "a set of options" to me. Maybe "option_status"?

Reply:
Changed to `option_command`.